src/connections/DelayedJob.ts

```diff
@@ -66,38 +59,32 @@ export const useDelayedJobs = () => {
   return connection;
 };
 
-export const triggerBulkUpdate = (jobs: DelayedJobData[]) => {
-  return bulkUpdateJobs({ body: { data: jobs } });
-};
+export const triggerBulkUpdate = (jobs: DelayedJobData[]) => bulkUpdateJobs({ body: { data: jobs } });
 
-const bulkUpdateJobsSelector = (store: ApiDataStore) => {
-  const bulkUpdateState = store.delayedJobs || {};
-  return {
-    isLoading: bulkUpdateJobsIsFetching(store),
-    hasLoadFailed: bulkUpdateJobsFetchFailed(store) != null,
-    response: bulkUpdateState.response
-  };
-};
+const bulkUpdateJobsSelector = (store: ApiDataStore) => ({
+  isLoading: bulkUpdateJobsIsFetching(store),
+  hasLoadFailed: bulkUpdateJobsFetchFailed(store) != null,
+  response: store.delayedJobs
+});
 
 const bulkUpdateJobsConnection: Connection<DelayedJobsConnection, { jobs: DelayedJobData[] }> = {
-  load: async (connection, { jobs }) => {
+  load: (connection, { jobs }) => {
     const isLoaded = connectionBulkUpdateIsLoaded(connection);
     if (!isLoaded) {
-      await bulkUpdateJobs({ body: { data: jobs } });
+      bulkUpdateJobs({ body: { data: jobs } });
     }
   },
 
   isLoaded: state => !state.isLoading,
 
-  selector: createSelector([store => bulkUpdateJobsSelector(store)], ({ isLoading, hasLoadFailed, response }) => ({
+  selector: createSelector(bulkUpdateJobsSelector, ({ isLoading, hasLoadFailed, response }) => ({
     isLoading,
     hasLoadFailed,
     response
```

Comment:
This seems like it should be throwing an error in the TS compiler... Maybe I'm missing something, but this shape does not seem to match `DelayedJobsConnection`. 

A couple of thoughts here:
* Typically, each connection will have its own shape. That's not required, but it's normal
* I've found in the past that properties like `isLoading`, `hasLoadFailed` and `response`, while seeming convenient when looking at a single connection, are actually a disaster down the road. The reason being that if we follow such generic conventions for individual connections, then when a component ends up using 2 or more connections, it gets really awkward to juggle all the `isLoading`s that each connection returns. Instead, I think it's better to be super specific with the properties of the connection shape. So something more like `bulkUpdateJobsIsLoading` and `bulkUpdateJobsHasFailed`.

As for `response` - this appears to be just returning all the delayed jobs like the other connection does. Are these used in the same component? Perhaps a better approach would be to combine them into a single connection. I think of connections as being responsible for providing a given set of data from the redux cache, and a set of functions for operating on that data. It would be totally reasonable for this `bulkUpdateJobs` call to be triggered from a function that's included on the other connection. Or, the bulkUpdate function could be simply exposed by this file, and then we simply rely on the original connection to provide the `delayedJobs` from the store. One of the primary wins of this system is that a connection returns data that got cached in the store _regardless of where it came from_. So if one call does the initial fetch, and another call modifies that data, as long as the data ends up back in the store under the same ID, the component that relies on it will rerender when it gets updated.

Reply:
Finally joined both connections. 
Tested and is working.